dotcom-rendering/src/components/FlexibleGeneral.stories.tsx

```diff
@@ -203,7 +203,7 @@ export const FourSublinkSplash: Story = {
 	},
 };
 
-const liveUpdatesCard = {
+export const liveUpdatesCard = {
```

Comment:
Not a storybook expert, but it looks to me like adding this `export` is causing it to consider this as its own story (basically a blank one), which I don't think is the intention?
<img width="1224" alt="image" src="https://github.com/user-attachments/assets/2a87ae9a-99ba-4e12-99f4-ab60f91de6f6" />

Unless there's somewhere else we can put the export, unfortunately we might need to redeclare the same const in the flexible special stories.

Reply:
Ahh good spot. I'll redeclare it.